src/gromacs/nblib/molecules.h

```diff
@@ -97,10 +96,10 @@ class Molecule
     template<typename T>
     Molecule& addAtom(const T& atomName, AtomType const& atomType) = delete;
 
-    void addHarmonicBond(HarmonicType harmonicBond);
+    void addHarmonicBond(const HarmonicType& harmonicBond);
 
     // TODO: add exclusions based on the unique ID given to the atom of the molecule
-    void addExclusion(const int atomIndex, const int atomIndexToExclude);
+    void addExclusion(const int& atomIndex, const int& atomIndexToExclude);
```

Comment:
I think the reference is not needed here. It actually adds overhead compared to a call by value, since an address must be computed to the variable.